decidim-core/app/helpers/decidim/widget_urls_helper.rb

```diff
@@ -2,8 +2,8 @@
 module Decidim
   module WidgetUrlsHelper
     def embed_modal_for(url)
-      js_embed_code = "#{content_tag(:script, '', src: url)}"
-      embed_code = "#{content_tag(:noscript, content_tag(:iframe, '', src: url.gsub(".js", ".html"), frameborder: 0, scrolling: "vertical"))}"
+      js_embed_code = content_tag(:script, "", src: url).to_s
```

Comment:
If I understand it correctly, string interpolation just implicitly calls `to_s` on every interpolated expression and the concatenates everything. So interpolating a single expression is equivalent to just calling `to_s` on it.

However, the auto-correction cop might be missing some subtlety and could have a bug, of course. So it never hurts to check this manually.